group17-s2/a4/c.txt

```diff
@@ -0,0 +1,10 @@
+The capability to read files and directories needs to be given to bash and to cat. Otherwise cp as well as iterating through directories doesn't work (bash), neither does opening the dependency files (cat). The capability is set by
```

Comment:
```suggestion
The capability to read files and directories needs to be given to the shell (bash in this case) and to cat. Otherwise cp as well as iterating through directories doesn't work (bash), neither does opening the dependency files (cat). The capability is set by
```